ui/components/Dashboard/index.js

```diff
@@ -35,6 +34,22 @@ import getWidgets from './widgets/getWidgets';
 import { tabsClasses } from '@mui/material';
 import { useSelector } from 'react-redux';
 
+const MESHERY_COLORS = {
+  primary: '#00B39F',
```

Comment:
We'll need to avoid redefining colors that are already in the design system, Sistent, but instead we need to reference that theme.

Use of glass-styled buttons and other components could be great! We'll want to take their consideration into context of the larger design system and propose when and how they should be used. Then, propagate that use across other components in accordance with this style's agreed upon use, so that we offering a conSistent UX to users.

Please explore, consider, and propose what this new styling might mean, so that we can promote its uniform use.